libraries/zmusic/mididevices/music_fluidsynth_mididevice.cpp

```diff
@@ -140,8 +140,12 @@ class FluidSynthMIDIDevice : public SoftSynthMIDIDevice
 #define FLUIDSYNTHLIB1	"libfluidsynth.1.dylib"
 #define FLUIDSYNTHLIB2	"libfluidsynth.2.dylib"
 #else // !__APPLE__
+#ifndef FLUIDSYNTHLIB1
 #define FLUIDSYNTHLIB1	"libfluidsynth.so.1"
+#endif
+#ifndef FLUIDSYNTHLIB2
 #define FLUIDSYNTHLIB2	"libfluidsynth.so.2"
+#endif
```

Comment:
What's the purpose of this change? If it's about providing custom library names, then why is it for FluidSynth only?

Reply:
Because it is only needed for these.